src/components/Button/Button.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { ReactComponent as PlusIcon } from 'assets/plus-icon.svg';
+import './Button.scss';
+
+const Button = React.forwardRef(({ type, icon, children, link, onClick, disabled }, ref) => {
+  const hasIcon = (icon) => icon ? <PlusIcon className='button__plus-icon'/> : null;
+
+  const rootURL = new RegExp('huoa.org/')
+  if (rootURL.test(link)) {
+    let match = rootURL.exec(link);
+    link = link.slice(match.index + 8)
```

Comment:
can you not do something like:
```
link = link.slice(rootURL.exec(link).index + 8)
```
?

Reply:
Yeap, that would save a line. The match variable was created more for readability so that a future dev can see what I'm doing. I'll go ahead and make the change.